WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -302,6 +386,7 @@ public void onSaveInstanceState(Bundle outState) {
         outState.putBoolean(KEY_IS_SHOWING_INSTALL_FIRST_PLUGIN_CONFIRMATION_DIALOG,
                 mIsShowingInstallFirstPluginConfirmationDialog);
         outState.putBoolean(KEY_IS_SHOWING_AUTOMATED_TRANSFER_PROGRESS, mIsShowingAutomatedTransferProgress);
+        outState.putBoolean(KEY_IS_SHOWING_DOMAIN_CREDIT_CHECK_PROGRESS, mIsShowingDomainCreditCheckProgress);
```

Comment:
The `mIsShowingDomainCreditCheckProgress` is never set to true, we may want to consider checking `mCheckingDomainCreditsProgressDialog!=null && mCheckingDomainCreditsProgressDialog.isShowing()` instead and use a local variable in the onCreate method. Wdyt?

Reply:
Missed that one 🤦‍♂️ Yes, local variable will work in this case :)